pkg/dockerregistry/server/manifestschema1handler.go

```diff
@@ -51,7 +52,7 @@ type manifestSchema1Handler struct {
 
 var _ ManifestHandler = &manifestSchema1Handler{}
 
-func (h *manifestSchema1Handler) FillImageMetadata(ctx context.Context, image *imageapi.Image) error {
+func (h *manifestSchema1Handler) FillImageMetadata(ctx context.Context, image *imageapiv1.Image) error {
```

Comment:
pre-existing, but mutation of args always rubs me the wrong way.

Reply:
will capture this as follow up for @legionus